netebpfext/net_ebpf_ext_sock_addr.c

```diff
@@ -1201,13 +1174,12 @@ _net_ebpf_ext_sock_addr_is_connection_locally_redirected_by_others(
 
 static NTSTATUS
```

Comment:
missing ```_Must_inspect_result_``` annotation? This function returns the status from a system api (```FwpsAcquireWritableLayerDataPointer```).  Are there scenarios where this failure need not be checked by the caller?